config/routes.rb

```diff
@@ -4,6 +4,10 @@
 
   # You can have the root of your site routed with "root"
   root 'home#index'
```

Comment:
You should take advantage of Rails resource routing to declare the common routes for a controller.

```ruby
Rails.application.routes.draw do
  root 'home#/index'

  resources :games, only: [:index, :show, :create, :update]
end
```